crm_project_task/wizards/crm_create_task.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2023 Moduon Team S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl-3.0)
+
+
+from odoo import _, fields, models
+from odoo.exceptions import UserError
+
+
+class CrmCreateTAsk(models.TransientModel):
+    _name = "crm.create.task"
+    _description = "Wizard to create task"
+
+    lead_id = fields.Many2one("crm.lead")
+    name_task = fields.Char()
+    description = fields.Html()
```

Comment:
The default value should be the lead's description:

![imagen](https://github.com/OCA/crm/assets/973709/8d05ca4e-d117-4dcd-9d50-79c77da7c459)

Reply:
@yajo the description of the task don't have and relation with the internal notes of the lead, explanation:

- Internal notes of the lead are comercial notes, only for salesman of the lead, shouldn't be propagated

- Description of the task is what a "project uset" should do/make and nothing about the sales

So this is in fact, intended 👍🏼 

Thank you!